docs/USAGE.md

```diff
@@ -170,19 +170,27 @@ You can download our pre-computed embeddings here.
+| other_polymorphic | `python src/br/analysis/run_features.py --save_path "/outputs_other_polymorphic/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/other_polymorphic" --sdf True --dataset_name "other_polymorphic" --debug False` |
+| other_punctate    | `python src/br/analysis/run_features.py --save_path "/outputs_other_punctate/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/other_punctate" --sdf False --dataset_name "other_punctate" --debug False`         |
+| pcna              | `python src/br/analysis/run_features.py --save_path "/outputs_pcna/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/pcna" --sdf False --dataset_name "pcna" --debug False`                                       |
+
+2. To run analysis like latent walks and archetype analysis on the embeddings and trained models, run the following commands.
+
+| Dataset           | Benchmarking features                                                                                                                                                                                                                                                                                         |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| cellpack          | `python src/br/analysis/run_analysis.py --save_path "./outputs_cellpack/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/cellpack" --dataset_name "cellpack" --run_name "Rotation_invariant_pointcloud_jitter_cellpack" --sdf False`                                    |
+| npm1_perturb      | `python src/br/analysis/run_analysis.py --save_path "./outputs_npm1_perturb/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/npm1_perturb" --dataset_name "npm1_perturb" --run_name "Rotation_invariant_pointcloud_jitter_npm1_perturb" --sdf True`                     |
+| npm1              | `python src/br/analysis/run_analysis.py --save_path "./outputs_npm1/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/npm1" --dataset_name "npm1" --run_name "Rotation_invariant_pointcloud_jitter_npm1" --sdf True`                                                     |
+| other_polymorphic | `python src/br/analysis/run_analysis.py --save_path "./outputs_other_polymorphic/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/other_polymorphic" --dataset_name "other_polymorphic" --run_name "Rotation_invariant_pointcloud_jitter_other_polymorphic" --sdf True` |
```

Comment:
run name is `Rotation_invariant_pointcloud_SDF`

Reply:
Resolved in [5b7daed](https://github.com/AllenCell/benchmarking_representations/pull/70/commits/5b7daedaf4eca66999ac25986f2e7da3d3bed009)